textbook/13/3/HypothesisTesting_3_TwoSample.ipynb

```diff
@@ -175,30 +175,48 @@
    "id": "7d59dfef",
    "metadata": {},
    "source": [
-    "We will focus in this example on BMI. Below are boxplots for the two diabetes status groups."
+    "We will focus in this example on BMI. Below are **boxplots** for the two diabetes status groups."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "087837b7-3f0a-4bed-8b5c-d9db96118016",
+   "metadata": {},
+   "source": [
+    ":::{note}\n",
```

Comment:
Maybe we can word this as a reminder since they should have seen this previously